src/vivarium/framework/results/stratification.py

```diff
@@ -38,6 +38,13 @@ class Stratification:
     mapper: Optional[Callable[[Union[pd.Series[str], pd.DataFrame]], pd.Series[str]]] = None
     is_vectorized: bool = False
 
+    def __str__(self) -> str:
+        return (
+            f"Stratification '{self.name}' with sources {self.sources}, "
+            f"categories {self.categories}, mapper {self.mapper.__name__}, "
+            f"and is_vectorized {self.is_vectorized}"
```

Comment:
I think is vectorized can be left out of the str? It's not super important, as it's really just there to help vivarium know how to use the mapper.